### DISCUSSION TITLE
Best way to determinate color / format of a video using libmpv

### DISCUSSION THREAD

**Discussion by skrew** [mpv-player/mpv#14024] [Q&A]
### Important Information

Provide following Information:
- mpv 0.38 (libmpv)
- Apple platform using gpu-next

Hi, what is the best practice to detect if a video is in Dolby vision, HDR10+, HDR or SDR ?
Actually i'm using `colormatrix-primaries` and `colormatrix` properties but this is not 100% effective.

Any recommendations ? Thanks.

**Comment by kasper93** [comment]
`video-params` have this information.

Examples for Dolby Vision:
```
[DV]
profile-cond=get("video-params/colormatrix") == "dolbyvision" and get("video-params/colorlevels") ~= "full"
profile-restore=copy
...
```

```
[DV-P5]
profile-cond=get("video-params/colormatrix") == "dolbyvision" and get("video-params/colorlevels") == "full"
profile-restore=copy
...
```

For HDR10+, you will get `video-params/scene-max-*` populated. If not it means it is HDR10.

https://github.com/mpv-player/mpv/issues/14014 may improve a bit the situation of format detection, because I will add this information to `track-list` property. At least if it is available from container.
